# The "-no-opaque-pointers" commandline parameter has been added by clang-15. Once clang-15 is available for windows, this patch can be removed.

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1415,10 +1415,6 @@ config("default_warnings") {
     if (treat_warnings_as_errors) {
       cflags += [ "/WX" ]
     }
-    if (fatal_linker_warnings) {
-      arflags = [ "/WX" ]
-      ldflags = [ "/WX" ]
-    }
     defines = [
       # Without this, Windows headers warn that functions like wcsnicmp
       # should be spelled _wcsnicmp. But all other platforms keep spelling
@@ -1542,8 +1538,6 @@ config("default_warnings") {
                         default_toolchain == "//build/toolchain/cros:target")) {
         # TODO(https://crbug.com/1316298): Re-enable once test failure is figured out
         cflags += [
-          "-Xclang",
-          "-no-opaque-pointers",
         ]
       }
 
